internal/pkg/cli/dialog/use_template.go

```diff
@@ -282,7 +283,7 @@ func (d *useTmplInputsDialog) announceStep(step *input.StepExt) error {
 	step.Announced = true
 
 	// Print description
-	d.Printf("%s\n%s", step.NameForDialog(), step.DescriptionForDialog())
+	d.Printf("%s\n%s", step.NameForDialog(), markdown.Render(step.DescriptionForDialog(), 80, 0))
```

Comment:
Formátování markdownu v `step.dialogDescription` (který se vezme z `step.DescriptionForDialog`). Na tohle bohužel nemůžu napsat e2e test, protože se to vypisuje v interaktivním módu commandu použití šablony a to neumíme otestovat. Ale aspoň jsem to přidal navíc do commandu na describe šablony (níž) a tam ten e2e test je.

Reply:
Tady je to ozkoušený lokálně:
![CleanShot 2022-06-28 at 15 26 23](https://user-images.githubusercontent.com/215660/176190360-de673402-5047-4c70-a176-29bfb8924392.jpg)